docs/synthetics/browser-monitor.md

```diff
@@ -11,9 +11,13 @@ The browser monitor executes the configured script in a Chrome browser. It recor
 * **Interval** - Monitor execution interval.
 * **Locations** - List of locations to run the monitor.
 
-### Script
+### URL/Script
 
-The browser monitor scripts are Node.js scripts that control a headless Chrome browser and use [Google Puppeteer](https://github.com/puppeteer/puppeteer#puppeteer) framework to drive the browser. For every run, the monitor will invoke the `testScript()` method with Puppeteer [Page](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#class-page) object as a parameter. The script content should be inside the `testScript()` method. For more information on specific use cases, refer to the `Browser Examples` section while creating a browser monitor. Use Node.js [assert](https://nodejs.org/api/assert.html) API to check if the values in the page match your requirements. If any assertion fails, the system declares the run as a failure. Monitor run logs, shown in Sematext Run details page, contain failure details.
+To monitor a single website, you can directly configure the URL of the website to be monitored. To monitor a user journey across multiple pages or perform actions on your website, you configure a script to simulate the user actions.
+
+When you configure the URL of the website, the browser monitor will load the URL in the Google Chrome browser and takes a screenshot of the website once the page load is complete.
+
+The user journey scripts are Node.js scripts that control a headless Chrome browser and use [Google Puppeteer](https://github.com/puppeteer/puppeteer#puppeteer) framework to drive the browser. For every run, the monitor will invoke the `testScript()` method with Puppeteer [Page](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#class-page) object as a parameter. The script content should be inside the `testScript()` method. For more information on specific use cases, refer to the `Browser Examples` section while creating a browser monitor. Use Node.js [assert](https://nodejs.org/api/assert.html) API to check if the values in the page match your requirements. If any assertion fails, the system declares the run as a failure. Monitor run logs, shown in the run details page, contain failure details.
```

Comment:
and use Google Puppeteer

=>

and use the Google Puppeteer

Reply:
Can you put a newline before the "Use Node.js assert..." paragraph. We need to break up this bit chunk of text with some whitespace.